### ISSUE TITLE
puppet exported resources support?

### ISSUE THREAD

**Issue by stig** [andyshinn/puppet-remote_syslog2#6]
Does this module have support for exported resources? I'm hoping to be able to configure the log files to be shipped in the manifest of the application that creates it. 

Thanks :-)

**Comment by andyshinn** [comment]
Nope, it doesn't use any exported resources. That is probably a much better way of handling log files and would likely be combined with #5. Unfortunately, this module was for a specific Puppet project and I am not longer working on that project, so my dedicated time to this module is near zero (until I have a need or want to use Puppet again).

Docs on exported resources for my future references: https://docs.puppetlabs.com/guides/exported_resources.html

**Comment by stig** [comment]
Thanks!
